data/json/snippets/e_logs.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "type": "snippet",
+    "category": "local_files_simple",
+    "//": "mostly work of random people: chat logs, SMSes, popular photos and videos that people share, or something they can film by themselves. ",
+    "text": [
+      {
+        "id": "local_files_simple_1",
+        "text": "system/storage/Download/EAS_apr_25,_15.57.mp3  \n\nThe file begins with complete silence, but then, a tired voice breaks the stillness:  \n\n\"My name is Clayton Gore, I am the head of [unintelligible].  \n\nI'm sorry, i WAS the head of [unintelligible]. \n\nI talk to you using the remains of Emergency Alert System, with everything we have left here. \n\nThis is the final message. \n\n\nThe government has fallen.  \n\nThere is nothing left.  \n\n\nNo help is coming. Use your weapons, your clubs, your fists and your teeth to survive this hard time.  \n\nNo one will arrest you.  \n\nNo one will help you.  \n\n\nYou're on your own.  \n\n\n\nEnd of transmission.\""
+      },
+      {
+        "id": "local_files_simple_2",
+        "text": "system/storage/Camera/IMG_202X0401_111567.jpg  \n\nHigh quality photo of an angry civil mob that suppressed the line of shielded policemans.  There is few dead people on the edge of a photo, one of which lay with chopped head."
+      },
+      {
+        "id": "local_files_simple_3",
+        "text": "apps/myMail  \n\nby:dnance@mymail.com  \nto:All  \nSubject: Tragic Loss of Rob Dixon  \n\nDear Colleagues,  \n\nIt is with a heavy heart that I have to inform you all of the passing of the head of supply chain department, Rob Dixon.  He was at the office late last night trying to ensure that everything was under control when he was caught in one of the riots that have been happening all over the city.  Despite all efforts, Rob did not make it.  \nRob was more than just a boss to us, he was a friend and mentor to many of us.  Rob always put the company and its employees first and he will be deeply missed.  \nPlease keep Rob’s family and loved ones in your thoughts and prayers during this difficult time.  The company is working closely with the authorities to ensure that Rob’s death is investigated fully.  \n\nIn the meantime, all of Rob's responsibilities will be temporarily handled by James Sullivan, his senior assistant.  If you have any urgent matters, please reach out to James.  \nWe will provide updates as they become available and further details on arrangements to celebrate Rob's life.  \n\nSincerely,\nDahila Nance\nHead of HR department."
```

Comment:
```suggestion
        "text": "apps/myMail  \n\nby:dnance@mymail.com  \nto:All  \nSubject: Tragic Loss of Rob Dixon  \n\nDear Colleagues,  \n\nIt is with a heavy heart that I have to inform you all of the passing of the head of the supply chain department, Rob Dixon.  He was at the office late last night trying to ensure that everything was under control when he was caught in one of the riots that have been happening all over the city.  Despite all the efforts, Rob did not make it.  \nRob was more than just a boss to us, he was a friend and a mentor to many of us.  Rob always put the company and its employees first and he will be deeply missed.  \nPlease keep Rob’s family and loved ones in your thoughts and prayers during this difficult time.  The company is working closely with the authorities to ensure that Rob’s death is fully investigated.  \n\nIn the meantime, all of Rob's responsibilities will be temporarily handled by James Sullivan, his senior assistant.  If you have any urgent matters, please reach out to James.  \nWe will provide updates as they become available and further details on arrangements to celebrate Rob's life.  \n\nSincerely,\nDahila Nance\nHead of HR department."
```

Reply:
I sure wish Github did it's job well and highlighted the changes. They are there... somewhere, trust me with this one.